src/main/java/com/github/dockerjava/api/model/ContainerHostConfig.java

```diff
@@ -0,0 +1,46 @@
+package com.github.dockerjava.api.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+/**
+ * Used in {@link Container}
+ *
+ * @see Container
+ * @author Kanstantsin Shautsou
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ContainerHostConfig {
+    @JsonProperty("NetworkMode")
+    private String networkMode;
+
+    public String getNetworkMode() {
+        return networkMode;
+    }
+
+    /**
+     * @see #networkMode
+     */
+    public ContainerHostConfig withNetworkMode(String networkMode) {
```

Comment:
They required for testing either there is no way natively create an object with data.